pocs/utils/database.py

```diff
@@ -57,9 +69,10 @@ def __init__(self, db='panoptes', host='localhost', port=27017, connect=False):
             'weather',
         ]
 
-        db_handle = getattr(self._client, db)
+        # Create an attribute on the client with the db name.
+        db_handle = self._client[db]
 
-        # Setup static connections to the collections we want
+        # Setup static connections to the collections we want.
```

Comment:
Does the `getattr(db_handle, collection)` create the collection? Or is that lazy too (i.e. maybe pymongo doesn't do anything until we try to read or write from the collection)?

No need to change this, but if the operation isn't lazy, and we wanted it to be, we could instead add a `__getattr__` method that validates that the attr is a member of `self.collections` before doing the setattr.

Reply:
It's lazy and happens upon first read/write.

Reply:
I'm just going to leave as is for now. Probably a few more things we could do to improve overall but nothing necessary right now.